src/components/TextInput/BaseTextInput/index.js

```diff
@@ -250,7 +250,7 @@ function BaseTextInput(props) {
     return (
         <>
             <View
-                style={styles.pointerEventsNone}
+                style={[styles.pointerEventsNone, ...props.containerStyles]}
```

Comment:
Moved `containerStyles` to main `View` wrapper caused an issue https://github.com/Expensify/App/issues/40519, bcoz `containerStyles` has max height restriction which specifically required only for TextInput Due to this change the error message overflows and overlaps with bottom content.